BEIMA.Backend/DeviceFunctions/GetDevice.cs

```diff
@@ -32,49 +33,49 @@ public static class GetDevice
         {
             log.LogInformation("C# HTTP trigger function processed a device get request.");
 
-            // Process as device GET request for retrieving device information.
-            if (string.Equals(req.Method, "get", StringComparison.OrdinalIgnoreCase))
```

Comment:
Not sure why the file is showing everything as changed. All that was removed was line 36 where we check if the request is a get request as that is already handled by the Http Trigger

Reply:
It's probably because the tabbing was changed.